City-of-Helsinki/drupal-helfi-platform-config
# UHF-3922: Added openid configs

# TASKNAME [UHF-3922](https://helsinkisolutionoffice.atlassian.net/browse/UHF-3922)
Changes the location of tunnistamo button on login page

## What was done
* Changed tunnistamo buttons location to above instead of below

## How to install
* Make sure your instance is up and running on latest dev branch.
    * `git pull origin dev`
    * `make fresh`
* Update the Helfi Platform config
    * `composer require drupal/helfi_platform_config:dev-UHF-3922-user-login-styling`
* Run `make drush-updb drush-cr`

## How to test
* In your instance go to /user/login and check that the tunnistamo button is before the normal login fields

## Other PRs
* https://github.com/City-of-Helsinki/drupal-hdbt/pull/281
